### PR TITLE
Misc Mods/Person/Packs/Drama

### PR DESCRIPTION
**Pull request by antoinegag**

### CODE REVIEW

File: data/author [hunk 1]
```diff
@@ -19,11 +20,13 @@ Clipboard:Shadowfacts
 ComputerCraft:dan200
 Crackpack:FTB
 Crash Landing:Iskandar
+CraftTweaker:Jared
```
Comment (JamiesWhiteShirt) [inline data/author]: Move one line up

File: data/author [hunk 2]
```diff
@@ -47,10 +50,13 @@ JABBA:ProfMobius
 JEI:mezz
 JourneyMap:techbrew
 Just Enough Dimensions:masa
+JustNuggets:DeflatedPickle
 LatBlocks:LatvianModder
 MatterOverdrive:SimeonRadivoev
+Melon Golem:Tamaized
```
Comment (JamiesWhiteShirt) [inline data/author]: One line down

File: data/person [hunk 3]
```diff
@@ -12,8 +15,10 @@ cpw
 CrazyPants
 dan200
 Darkhax
+DeflatedPickle
 Dinnerbone
 direwolf20
+Drullkus
```
Comment (JamiesWhiteShirt) [inline data/person]: One line down

File: data/person [hunk 4]
```diff
@@ -23,7 +28,9 @@ Etho
 Glitchfiend
 greenphelm
 HellFirePVP
+HyperionNexus
 iChun
+Jared
```
Comment (JamiesWhiteShirt) [inline data/person]: 3 lines down

File: data/person [hunk 5]
```diff
@@ -53,6 +61,7 @@ some guy on %community
 some guy on reddit
 SpitefulFox
 techbrew
+Tamaized
```
Comment (JamiesWhiteShirt) [inline data/person]: One line up

**Review by JamiesWhiteShirt — CHANGES_REQUESTED** [review]

File: data/author (hunk 1 above)
Comment (jaredlll08) [inline data/author]: Not sure if I should feel honored or not >.>

**Review by JamiesWhiteShirt — APPROVED** [review]

File: data/author (hunk 1 above)
Comment (antoinegag) [reply data/author]: You should

**Comment by Gaelan** [comment]
TIL I gave you a commit bit

**Pull request merged**